packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -67,6 +68,9 @@ import 'runtime.dart';
 ///  * The [Scaffold]'s floating action button position and animation features
 ///    are not supported.
 ///
+///  * [DropdownButton] takes a list of items object with the key of params 
+///    names and value of the param values to represent [DropdownMenuItem].
```

Comment:
I'm not sure exactly what this is trying to say but it doesn't seem to match the code — it seems to be a list of objects, and in each object there's zero or more of `onTap`, `value`, `enabled`, `alignment`, and `child`? (Not sure which are required and which are optional, but we should document that too.)

Reply:
Updated the comment.